community/samples/serving/helloworld-deno/main.ts

```
@@ -0,0 +1,10 @@
+import { serve } from "./deps.ts";
+
+const PORT = 8080;
```

Do not hardcode  the value of PORT. Use the $PORT environment variable and if not present fallback to an hardcoded number (8080)

---

Using an env var.